src/nyx/grayscale_tiff.h

```diff
@@ -165,10 +174,10 @@ class NyxusGrayscaleTiffTileLoader : public AbstractTileLoader<DataType>
             case 8:
             case 16:
             case 32:
-                loadTile<float>(tiffTile, tileDataVec);
+                loadTile_real_intens <float> (tiffTile, tileDataVec);

```

Comment:
can we do something like 
```
if (image_options_are_present) {
  loadTile_real_intens(...)
} else {
  loadTile(...)
}
```
That way, if we are not dealing with already scaled image (which is not the common case, we don't spend tile inspecting each pixel value while copying.

```

Reply:
Floating point image options (in the form of expected minimum, maximum, and equivalent integer range) are always present, aren't they? And doing your suggested way, we'd need to do a separate pre-scan through an image to figure out its intensity range, that is also a time expense.